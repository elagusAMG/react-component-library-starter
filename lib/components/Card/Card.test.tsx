import { expect, test, describe } from 'vitest';

import { render } from '@testing-library/react';

import { Card } from './Card';

describe('Card', () => {
  test('should match snapshot', () => {
    const { container } = render(<Card title="Card Title" description="This is a card" />);
    expect(container).toMatchSnapshot();
  });
});
