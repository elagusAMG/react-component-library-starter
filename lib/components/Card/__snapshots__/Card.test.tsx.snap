// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card > should match snapshot 1`] = `
<div>
  <div
    class="ui"
  >
    <div
      class="ui-bg-white ui-rounded-lg ui-shadow-lg ui-overflow-hidden sm:max-w-xs lg:max-w-sm xl:max-w-md"
    >
      <div
        class="ui-px-6 ui-py-4"
      >
        <h2
          class="ui-font-sans ui-font-bold ui-text-xl ui-mb-2 ui-text-pink"
        >
          Card Title
        </h2>
        <p
          class="ui-bg-white ui-border-none ui-rounded-none ui-font-sans ui-text-gray-dark ui-text-base"
        >
          This is a card
        </p>
      </div>
    </div>
  </div>
</div>
`;
